scripts/util/update-files.js

```diff
@@ -58,13 +60,25 @@ const updatePackageJsonVersion = async (version) => {
 
 }
 
+export const updateEnvironmentManifest = async (version, environment = 'staging') => {
+  const environmentsFile = './infrastructure/environments.yml';
+  const current = fs.readFileSync(environmentsFile, 'utf8')
+  let manifest = YAML.parse(current)
+  manifest[environment].wordpress = version;
+
+  let output = YAML.stringify(manifest);
+  fs.writeFileSync(environmentsFile, output);
```

Comment:
Thinking we don't need the output var here --- just pass YAML.stringify(manifest) into the teFileSync(

Reply:
true